src/rest/endpoints.rs

```diff
@@ -95,3 +96,60 @@ pub async fn run_scenario(
         .map(|_| StatusCode::OK)
         .map_err(|_| StatusCode::INTERNAL_SERVER_ERROR)
 }
+
+#[cfg(test)]
+mod test {
+    use crate::model::{Blob, BlobData, BlobTransaction, ContractName, Identity};
+
+    /*
+        curl -X POST --location 'http://localhost:4321/v1/tx/send/blob' \
+    --header 'Content-Type: application/json' \
+    --data '{
+        "identity": "ident",
+        "blobs": [
+            {
+                "contract_name": "contrat de test",
+                "data": []
+            }
+        ]
+    }'
+    */
+
+    #[test]
+    fn test_blob_tx_decode() {
+        let payload_json = r#"
+         {
+            "identity": "ident",
+            "blobs": [
+                {
+                    "contract_name": "contrat de test",
+                    "data": []
+                }
+            ]
+        }
+         "#;
+
+        let decoded: BlobTransaction = serde_json::from_str(payload_json).unwrap();
+
+        assert_eq!(decoded.identity, Identity("ident".to_string()));
+    }
+
+    #[test]
+    fn test_blob_tx_encode() {
+        let payload = BlobTransaction {
+            identity: Identity("tata".to_string()),
+            blobs: vec![Blob {
+                contract_name: ContractName("contract_name".to_string()),
+                data: BlobData(vec![]),
+            }],
+        };
+
+        let encoded = serde_json::to_string(&payload).unwrap();
+
+        assert_eq!(
+            encoded,
+            "{\"identity\":\"tata\",\"blobs\":[{\"contract_name\":\"contract_name\",\"data\":[]}]}"
+                .to_string()
+        );
+    }
```

Comment:
Not sure these tests are very relevant, they are testing the `serde` crate more than our code :/ 

And as soon as we will update these structs, we would break these tests. If we want to test serialisation I would make a single tests like that : 
```rust
let obj = BlobTransaction::default();
let encoded = encode(obj);
let decoded = decode(encoded);
assert_eq!(obj, decoded);
```

Reply:
No they are not really tests, it was just examples to call the api

Reply:
I transformed them into doc tests